charts/harvester-node-disk-manager/values.yaml

```diff
@@ -6,15 +6,15 @@ image:
   repository: rancher/harvester-node-disk-manager
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "v1.5.0-rc5"
+  tag: "v1.5.0"
```

Comment:
Just noticed this component is the only one requiring updating values.yaml.

Reply:
@Vicente-Cheng any insight that we want to specify this in the values.yaml?

Reply:
Actually, this is a historic issue. We could skip the update value.
The default tag would use the `.Chart.AppVersion`
REF: https://github.com/harvester/charts/blob/release/charts/harvester-node-disk-manager/templates/daemonset.yaml#L29